src/lib.rs

```diff
@@ -0,0 +1,53 @@
+use crate::run_context::RunContext;
+use anyhow::Result;
+use clap::Parser;
+use move_cli::Move;
+use std::path::PathBuf;
+
+mod cmd;
+mod run_context;
+mod run_move_cli;
+
+/// CLI frontend for the Move compiler and VM in Substrate
```

Comment:
let's add dots in our `///` comments
```suggestion
/// CLI frontend for the Move compiler and VM in Substrate.
```

Reply:
Will do in the next PR